.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,20 @@
+name: Pull request checks
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+jobs:
+  check-title:
+    name: Check title
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: naveenk1223/action-pr-title@v1.0.0
+        with:
+          regex: '^(ENSC-\d+ )?(chore|demo|deprecate|docs|feat|fix|perf|refactor|revert|style|test)(\(.+\))?: .+$'
+          max_length: 80
```

Comment:
Sticking with 80 instead of 60 for this?

Reply:
Good catch, fixed.